dato.config.js

```diff
@@ -1,23 +1,31 @@
 // @flow
 
-const filterByKeys = (data, filterKeys, areIncluded) =>
+const filterByKeys = (data, filterKeys, areIncluded, isTranslatedPart) =>
   Object.keys(data)
     .filter(key => (areIncluded ? filterKeys.includes(key) : !filterKeys.includes(key)))
-    .reduce(
-      (obj, key) => ({
+    .reduce((obj, key) => {
+      // to remove empty values from translations
+      if (isTranslatedPart && !data[key]) {
+        return {
+          ...obj,
```

Comment:
It's okay to `return obj;` here. We're not modifying the object, so there is no reason to create an identical one.